cmd/minikube/cmd/start_flags.go

```diff
@@ -525,9 +531,15 @@ func updateExistingConfigFromFlags(cmd *cobra.Command, existing *config.ClusterC
 	// validate the memory size in case user changed their system memory limits (example change docker desktop or upgraded memory.)
 	validateRequestedMemorySize(cc.Memory, cc.Driver)
 
```

Comment:
add a warning if user added this flag to non-hyperkit drivers, that
```
out.WarningT("Specifying extra disk is currently only supported for hyperkit driver, if you can contribute to add this feature please create a PR....")
```

Reply:
I added this check for flags on both create and update. If I were a user, I think I would expect using these flags for unsupported drivers would cause a failure since the option isn't supported. Would it be better for me to return an error instead of just logging a warning?

Reply:
since we are not passing the option to the driver if it doesnt suppor it, it wont do anything, I would be okay with ExitWith Usage or just warning. your call !

Reply:
I'll change this to exit with usage soon.